userbenchmark/torch-nightly/regression_detector.py

```diff
@@ -1,17 +1,20 @@
 from typing import Optional
 from ..utils import TorchBenchABTestResult, TorchBenchABTestMetric
 
+DEFAULT_REGRESSION_DELTA_THRESHOLD = 0.07
+
 def run(control, treatment) -> Optional[TorchBenchABTestResult]:
     control_env = control["environ"]
     treatment_env = treatment["environ"]
     control_metrics = control["metrics"]
-    treatment_metrics = control["metrics"]
+    treatment_metrics = treatment["metrics"]
```

Comment:
OH MY GOSH WHAT A GOOD CATCH.

--facepalm that i landed this code no wonder the deltas were always 0 :'(

Reply:
My bad, I didn't actually run the regression detection code when wrote this...